lib/std/c.zig

```diff
@@ -1360,6 +1360,20 @@ pub const KERN = switch (native_os) {
 pub const MADV = switch (native_os) {
     .linux => linux.MADV,
     .emscripten => emscripten.MADV,
+    .macos, .ios, .tvos, .watchos, .visionos => struct {
```

Comment:
Is this missing `driverkit`?

Reply:
Yeah but it's not really worth rerunning CI over. At some point I'll go over this file and fix all of these prongs together. This file is full of similar mistakes for all the other Apple targets besides `macos`.